lib/ueberauth/strategy/facebook.ex

```diff
@@ -65,8 +65,7 @@ defmodule Ueberauth.Strategy.Facebook do
         fetch_user(conn, client, config)
       end
     rescue
-      OAuth2.Error ->
-        set_errors!(conn, [error("invalid_code", "The code has been used or has expired")])
+      e -> set_errors!(conn, [error("get_token_error", e)])
```

Comment:
I am worry about this being a breaking change thou 🤔

Reply:
@yordis I have tested with this change in my system. Doesn't break for me (deployed this to my prod)

Reply:
e is a Struct / Map. set_errors! is able to handle it.

Reply:
Right, but the code was `invalid_code` and now it is `get_token_error`.

So what happens with the upstream consumers of it?

Reply:
@yordis , the problem here is that, its not `invalid_code` in most cases. If by `invalid_code` you mean code sent by facebook is invalid.

Reply:
I get that, my concern is not that your code does not make sense, it does, my concern is people relying on such code and this being a breaking change.

We should follow this provider style btw?
https://github.com/ueberauth/ueberauth_google/blob/3e7c31915d39226088dc17b9d90044b299bb662f/lib/ueberauth/strategy/google.ex#L45-L51

Thoughts?

Reply:
I will make the change

Reply:
This will probably not work, since ours is a `rescue` . The `GoogleAuth` is catching the error in `{:error, {error_code, error_description}}` particular format. As far as I remember, the error I got was different. I can try reproduce and post the error here. But I can't do it right now. I will take some time.